x-pack/plugins/ml/public/application/datavisualizer/stats_datagrid/components/field_data_expanded_row/number_content.tsx

```diff
@@ -0,0 +1,162 @@
+ */
+
+import React, { FC, Fragment, ReactNode, useEffect, useState } from 'react';
+import { EuiBasicTable, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { FieldDataCardProps } from '../../../index_based/components/field_data_card';
+import { kibanaFieldFormat } from '../../../../formatters/kibana_field_format';
+import { numberAsOrdinal } from '../../../../formatters/number_as_ordinal';
+import {
+  MetricDistributionChart,
+  MetricDistributionChartData,
+  buildChartDataFromStats,
+} from '../../../index_based/components/field_data_card/metric_distribution_chart';
+import { TopValues } from '../../../index_based/components/field_data_card/top_values';
+import { ExpandedRowFieldHeader } from '../expanded_row_field_header';
+
+const METRIC_DISTRIBUTION_CHART_WIDTH = 325;
+const METRIC_DISTRIBUTION_CHART_HEIGHT = 210;
+
+interface SummaryTableItem {
+  function: string;
+  display: ReactNode;
+  value: number | string | undefined | null;
+}
+
+export const NumberContent: FC<FieldDataCardProps> = ({ config }) => {
+  const { stats, fieldFormat } = config;
+
+  useEffect(() => {
+    const chartData = buildChartDataFromStats(stats, METRIC_DISTRIBUTION_CHART_WIDTH);
+    setDistributionChartData(chartData);
+  }, []);
+
+  const defaultChartData: MetricDistributionChartData[] = [];
+  const [distributionChartData, setDistributionChartData] = useState(defaultChartData);
+
+  if (stats === undefined) return null;
+  const { min, median, max, distribution } = stats;
+
+  const summaryTableItems = [
+    {
+      function: 'min',
+      display: (
+        <FormattedMessage id="xpack.ml.fieldDataCard.cardNumber.minLabel" defaultMessage="min" />
+      ),
+      value: kibanaFieldFormat(min, fieldFormat),
+    },
+    {
+      function: 'median',
+      display: (
+        <FormattedMessage
+          id="xpack.ml.fieldDataCard.cardNumber.medianLabel"
+          defaultMessage="median"
+        />
+      ),
+      value: kibanaFieldFormat(median, fieldFormat),
+    },
+    {
+      function: 'max',
+      display: (
+        <FormattedMessage id="xpack.ml.fieldDataCard.cardNumber.maxLabel" defaultMessage="max" />
+      ),
+      value: kibanaFieldFormat(max, fieldFormat),
+    },
+  ];
+  const summaryTableColumns = [
+    {
+      name: '',
+      render: (summaryItem: { display: ReactNode }) => summaryItem.display,
+    },
+    {
+      field: 'value',
+      name: '',
+      render: (v: string) => <strong>{v}</strong>,
+    },
+  ];
+
+  const summaryTableTitle = i18n.translate('xpack.ml.fieldDataCard.cardNumber.summaryTableTitle', {
+    defaultMessage: 'summary',
+  });
+  return (
+    <EuiFlexGroup direction={'row'} data-test-subj={'mlNumberSummaryTable'}>
+      <EuiFlexItem>
+        <ExpandedRowFieldHeader>{summaryTableTitle}</ExpandedRowFieldHeader>
+        <EuiBasicTable<SummaryTableItem>
```

Comment:
Is it possible to hide the empty table header row here? Otherwise you get a thin white row below the `SUMMARY` title:

![image](https://user-images.githubusercontent.com/7405507/102091247-7babee80-3e16-11eb-8739-d1e606b09eba.png)

Reply:
Talked to the EUI team and this is not possible. However, since we are changing the background color to match the background anyway, this should be less noticeable.

<img width="1466" alt="Screen Shot 2020-12-14 at 13 23 15" src="https://user-images.githubusercontent.com/43350163/102126060-308edd00-3e10-11eb-9f91-707f457cd939.png">